package.json

```diff
@@ -73,12 +73,13 @@
     "file-url": "2.0.2",
     "is-plain-object": "2.0.1",
     "jquery": "3.4.1",
-    "jsbloom": "git://github.com/duckduckgo/jsbloom.git#1.0",
+    "jsbloom": "git://github.com/duckduckgo/jsbloom.git#1.0.2",
```

Comment:
I had to fix a bug in jsbloom because it didn't allow to create two bloom filters in the same app: https://github.com/duckduckgo/jsbloom/releases/tag/1.0.2